onnxruntime/core/framework/data_transfer_manager.cc

```diff
@@ -51,4 +50,55 @@ Status DataTransferManager::CopyTensor(const Tensor& src, Tensor& dst, int exec_
                          dst.Location().device.ToString());
 }
 
+common::Status DataTransferManager::CopyTensors(const std::vector<IDataTransfer::SrcDstPair>& src_dst_pairs) const {
+  if (src_dst_pairs.empty())
```

Comment:
Should we log something here in case the caller made a mistake?

Reply:
We could be but it seems low value. It's an internal method currently only called from two places. If it was user facing I think a message would be worthwhile.